### ISSUE TITLE
Update page navigation

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#824]
As part of our design pass coming out of Accessibility feedback, we'd like to update the styling of the navigation at the bottom of the [Listings](https://sites-public-tj3gg4i5eq-uc.a.run.app/listings) page. The buttons should have a more rounded styling and the copy should be slightly tweaked (see attached screenshot).
![listing_navigation](https://user-images.githubusercontent.com/394908/145231251-af1f8ff8-eaee-4d98-9085-ae4a86f7710a.png)

Design is viewable in Figma (02.1).
![listing_navigation](https://user-images.githubusercontent.com/394908/145231161-24322606-30c9-4d98-a6e9-eef0ef141fa1.png)
